libshaderc/include/shaderc.hpp

```diff
@@ -67,6 +67,22 @@ class SpvModule {
     return shaderc_module_get_length(module_);
   }
 
+  // Returns the number of warnings generated during the compilation.
+  size_t GetNumWarnings() const {
```

Comment:
Perhaps for another CL, we should document (and test) the behaviour of the SpvModule getters when no compiled module exists.